rotkehlchen/typing.py

```diff
@@ -222,8 +222,25 @@ def get_address_type(self) -> Callable:
             return BTCAddress
         if self == SupportedBlockchain.KUSAMA:
             return KusamaAddress
-        # else
-        raise AssertionError('Invalid SupportedBlockchain value')
+        raise AssertionError(f'Invalid SupportedBlockchain value: {self}')
+
+    def ens_coin_type(self) -> int:
+        """Return the CoinType number according to EIP-2304, multichain address
+        resolution for ENS domains.
+
+        https://eips.ethereum.org/EIPS/eip-2304
```

Comment:
I don't like too many links and information duplication. Remove the rest of the links. The EIPS link is sufficient and has all the numbers and the spec.

Reply:
True, all this info and resources should be in the GitHub issue